docker-compose.yml

```diff
@@ -372,6 +374,8 @@ services:
     volumes: 
       - ./app:/usr/src/code/app
       - ./src:/usr/src/code/src
+      - appwrite-functions:/storage/functions:rw
```

Comment:
Why realtime needs access to this volumes? As a good practice we always strive to segment data so only the containers that MUST have access to it are granted with it.

Reply:
It's in the `builds` worker and not `realtime` 😄